pkg/operator/ceph/object/objectstore_test.go

```diff
@@ -301,40 +301,40 @@ func TestMockExecHelperProcess(t *testing.T) {
 func Test_createMultisite(t *testing.T) {
 	// control the return values from calling get/create/update on resources
 	type commandReturns struct {
-		realmExists         bool
-		zoneGroupExists     bool
-		zoneExists          bool
-		periodExists        bool
-		failCreateRealm     bool
-		failCreateZoneGroup bool
-		failCreateZone      bool
-		failUpdatePeriod    bool
+		realmExists             bool
+		zoneGroupExists         bool
+		zoneExists              bool
+		failCreateRealm         bool
+		failCreateZoneGroup     bool
+		failCreateZone          bool
+		failCommitConfigChanges bool
```

Comment:
TL;DR of changes in this file: no longer use raw `period get`/`period update --commit` to create multisite. Instead, use a mock `CommitConfigChanges`, validate that the function (unit tested elsewhere) is called, and control when it returns success/fail.